tools/README.md

```diff
@@ -101,11 +100,20 @@ address = 0x9df878E37C4B851bbe44841e72299F6f317C685C
 password-file = ./keys/pass-UTC--2019-11-22T13-50-19.904075843Z--9df878e37c4b851bbe44841e72299f6f317c685c
 ```
 
-This script does not take into account fees, therefore it must the no-fee PFS
+This script does not take into account fees, therefore it must use the no-fee PFS
 (as per the example configuration above).
 
-pylint
-------
+Before using the stress test for the first time, you need to prepare the three nodes given in the config file:
+1. Create three accounts (e.g. with `geth account new`).
+2. Create the config file.
+3. Uncomment the `input()` line in `stress_test_transfers.py` and start the script with the config file. Now the three nodes are running and you can use other scripts to interact with them.
+4. Use `mint.sh` to mint tokens for the nodes (see section above).
+5. Use `channel_with_minimum_balance.py` to create the channels and deposit tokens into the channels (see section above).
+6. Now you can press enter to proceed past the `input` and start the stress test (or remove the `input()` line and start the script, again).
+
+Going through these steps is cumbersome, but once you're done you can rerun the stress test any time without needing to repeat these steps. Only in case of failing stress tests, you might need to refill the channel balances with `channel_with_minimum_balance.py`.
```

Comment:
Added some small adjustments to the script [here](https://github.com/raiden-network/raiden/pull/5615).

I can also add the dummy configs, since I already have them in my local copy if you like.